src/Symfony/Component/Messenger/Bridge/AmazonSqs/Transport/AmazonSqsTransport.php

```diff
@@ -89,7 +98,7 @@ public function setup(): void
         }
     }
 
-    public function reset()
+    public function reset(): void
```

Comment:
revert, this is a BC break

Reply:
Thanks, never would have thought, but indeed, inheriting classes would be broken :-)